getting-started/introduction-to-aepedge.md

```diff
@@ -6,27 +6,29 @@ This documentation section is in the process of being moved - please see [https:
 
 ![](../.gitbook/assets/AEP_Edge_dataflow.png)
 
-## Key Benefits
+## Key benefits
 
-* Performance - Reduced number of network calls from your web/mobile applications to improve customer experience
-* Smaller footprint – Decrease in the number of extensions and dependencies to manage for mobile implementations
-* Simplified data collection – Standardize data collection with XDM, reducing developer complexity and maintenance
-* Privacy - Improved privacy controls and consent management
-* Server Side Forward – Take advantage of Launch Server Side rules and extensions to deliver data to 3rd parties
+* **Performance**: Reduced number of network calls from your web/mobile applications to improve customer experience.
+* **Smaller footprint**: Decrease the number of extensions and dependencies to manage for mobile implementations.
+* **Simplified data collection**: Standardize data collection with XDM, reducing developer complexity and maintenance.
+* **Privacy**: Improved privacy controls and consent management.
+* **Server-side forward**: Take advantage of Launch Server Side rules and extensions to deliver data to 3rd parties.
 
-## Adobe Experience Platform Edge Network Migration Considerations
+## Adobe Experience Platform Edge Network migration considerations
 
-The new Experience Platform Edge Network extension requires the usage of the latest Adobe Experience Platform SDKs, which for iOS is a new Swift-based SDK. This new Swift SDK for iOS is incompatible with existing Adobe solution extensions such as Adobe Campaign Standard, Adobe Campaign Classic, and Adobe Mobile Services. While updates to these solution extensions are in progress, organizations requiring multiple Adobe solutions should first check to ensure that a swift-compatible or Edge version of the extension exists. For the latest information on supported extensions see Current SDK Versions. 3rd party vendor extensions should continue to work as expected.
+The new Experience Platform Edge Network extension requires the usage of the latest Adobe Experience Platform SDKs. On iOS, the latest SDK is a new Swift-based SDK that is incompatible with existing Adobe solution extensions such as Adobe Campaign Standard, Adobe Campaign Classic, and Adobe Mobile Services. 
 
-Customers looking to adopt platform-based solutions such as Customer Journey Management or Customer Journey Analytics should consider adopting the new Experience Platform Edge Network extension to simplify sending XDM events to the platform.
+While updates to these solution extensions are in progress, organizations requiring multiple Adobe solutions should first check to ensure that a Swift-compatible or Edge version of the extension exists. For the latest information on supported extensions see [Current SDK Versions](). Third-party vendor extensions should continue to work as expected.
```

Comment:
link missing here, but I'm guessing that's to be filled later on

Reply:
The Current SDK Versions is this page here: /resources/upgrading-to-aep/current-sdk-versions